manifests/daemonset_openshift.yaml

```diff
@@ -0,0 +1,169 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: bridge-filtering
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - get
+      - list
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: bridge-filtering
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: bridge-filtering
+subjects:
+- kind: ServiceAccount
+  name: bridge-filtering
+  namespace: kube-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: bridge-filtering
+  namespace: kube-system
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: bridge-filtering-ds
+  namespace: kube-system
```

Comment:
That's a good point, maybe we can invent our own namespace for this?
Like `openshift-bridge-filtering-cni` ?

Reply:
openshift- namespaces are special. I would suggest we use bridge-filtering-cni. Since these are just yamls applied by user, they can tweak it if they don't like the name. I would not use openshift/cnv since this is not a part of the product. It should not be kube-system.

Reply:
`bridge-filter-2000` ?...

Joking aside, `bridge-filtering-cni` works for me.

Reply:
With the amount of bash, it could be be bridge-filter-9000

Reply:
[Renamed](https://github.com/rhrazdil/bridge-filtering-cni/pull/12/commits/fcce1cb720c2ec3b0020b36181ab3c82f90ebb0d) the namespace to `bridge-filtering-cni`